packaging/zip/Eluant.dll.config

```diff
@@ -0,0 +1,4 @@
+<configuration>
+  <dllmap os="linux" dll="lua51.dll" target="liblua5.1.so.0" />
```

Comment:
There is no way to do that in for a distribution independent package without including the lua library in the package as far as I can see. Maybe with some launcher which rewrites that file before launching but that won't work if the user decides to run it directly.